unit/suites/test_dbapi.py

```diff
@@ -0,0 +1,176 @@
+            host=self.srv.host,
+            port=self.srv.args['primary'])
+
+    def setUp(self):
+        # prevent a remote tarantool from clean our session
+        if self.srv.is_started():
+            self.srv.touch_lock()
+        self.con.flush_schema()
+
+        # grant full access to guest
+        self.srv.admin("box.schema.user.grant('guest', 'create,read,write,"
+                       "execute', 'universe')")
+
+    @classmethod
+    def tearDownClass(self):
+        self.con.close()
+        self.srv.stop()
+        self.srv.clean()
+
+    def test_rowcount(self):
+        con = self._connect()
+        try:
+            cur = con.cursor()
+            self.executeDDL1(cur)
+            dbapi20._failUnless(self,cur.rowcount in (-1,1),   # Bug #543885
+                'cursor.rowcount should be -1 or 0 after executing no-result '
+                'statements' + str(cur.rowcount)
+                )
+            cur.execute("%s into %sbooze values ('Victoria Bitter')" % (
+                self.insert, self.table_prefix
+                ))
+            dbapi20._failUnless(self,cur.rowcount in (-1,1),
+                'cursor.rowcount should == number or rows inserted, or '
+                'set to -1 after executing an insert statement'
+                )
+            cur.execute("select name from %sbooze" % self.table_prefix)
+            dbapi20._failUnless(self,cur.rowcount in (-1,1),
+                'cursor.rowcount should == number of rows returned, or '
+                'set to -1 after executing a select statement'
+                )
+            self.executeDDL2(cur)
+            dbapi20._failUnless(self,cur.rowcount in (-1,1),   # Bug #543885
+                'cursor.rowcount should be -1 or 0 after executing no-result '
+                'statements'
+                )
+        finally:
+            con.close()
+
+    @unittest.skip('Not implemented')
+    def test_Binary(self):
+        pass
+
+    @unittest.skip('Not implemented')
+    def test_STRING(self):
+        pass
+
+    @unittest.skip('Not implemented')
+    def test_BINARY(self):
+        pass
+
+    @unittest.skip('Not implemented')
+    def test_NUMBER(self):
+        pass
+
+    @unittest.skip('Not implemented')
+    def test_DATETIME(self):
+        pass
+
+    @unittest.skip('Not implemented')
+    def test_ROWID(self):
+        pass
+
+    @unittest.skip('Not implemented')
+    def test_Date(self):
+        pass
+
+    @unittest.skip('Not implemented')
+    def test_Time(self):
+        pass
+
+    @unittest.skip('Not implemented')
+    def test_Timestamp(self):
+        pass
+
+    @unittest.skip('Not implemented as optional.')
+    def test_nextset(self):
+        pass
+
+    @unittest.skip('To do')
+    def test_callproc(self):
+        pass
+
+    def test_setoutputsize(self):  # Do nothing
+        pass
+
+    @unittest.skip('To do')
+    def test_description(self):
+        pass
+
+    def test_dql_response(self):
```

Comment:
Nit: I would add those test within the commit, where the related functionality was added: I mean, where \<base connection\>.execute() was implemented. It also seems that those test are not much relevant to dbapi and it would be better to extract them to its own test suite.

Reply:
Replaced the tests in a corresponding commit with execute() implementation

Reply:
Okay, but you left those method in the test re dbapi.

Reply:
Sorry for the carelessness, I copied to the new test suite and forgot to remove from the old one. Fixed.